addon/mixins/axes.js

```diff
@@ -16,11 +16,61 @@ export default Ember.Mixin.create({
   graphicHeight: null,
   minXTicks: 3,
   minYTicks: 3,
-  tickSpacing: 50,
-
   minAxisValue: 0,
   maxAxisValue: 0,
 
+  /**
+   * We used to have only one option to set tick spacing for both x and y axes.
+   * We keep this attribute for backward compatibility.
+   * @type {Number}
+   * @deprecated since version 1.0
+   */
+  tickSpacing: 50,
+
+  /**
+   * Tick spacing on X axis.
+   * @type {Number}
+   */
+  tickSpacingX: null,
+
+  /**
+   * Tick spacing on Y axis.
+   * @type {Number}
+   */
+  tickSpacingY: null,
+
+  /**
+   * This will be used for all internal calculation of tick spacing on X axis.
+   * We set higher priority if the specific tickSpacingX's value is set.
+   * @type {Number}
+   */
+  innerTickSpacingX: Ember.computed('tickSpacingX', 'tickSpacing',
+    function() {
+      const tickSpacingX = this.get('tickSpacingX');
+      if (tickSpacingX !== null) {
```

Comment:
We can support it, but we don't need to. If people want to use the previous version `tickSpacing`, they will just don't set anything.